doc/doc.go

```diff
@@ -117,9 +117,13 @@ func outputs(list *ast.ObjectList) []Output {
 		if is(item, "output") {
 			name, _ := strconv.Unquote(item.Keys[1].Token.Text)
 
+			items := item.Val.(*ast.ObjectType).List.Items
 			var desc string
-			if c := item.LeadComment; c != nil {
-				desc = comment(c.List)
+			switch {
+			case description(items) != "":
+				desc = description(items)
+			case item.LeadComment != nil:
```

Comment:
Does it make more sense to maintain the same behaviour between `inputs` and `outputs`, and just add a note to the documentation?